src/bindings/python/flux/job/list.py

```diff
@@ -251,15 +251,18 @@ def fetch_jobs(self):
         )
 
     def jobs(self):
-        """Synchronously fetch a list of JobInfo objects from JobList query
+        """Synchronously fetch a list of jobs from JobList query
 
         If the Future object returned by JobList.fetch_jobs has not yet been
         fulfilled (e.g. is_ready() returns False), then this call may block.
-        Otherwise, returns a list of JobInfo objects for all jobs returned
-        from the underlying job listing RPC.
         """
         rpc = self.fetch_jobs()
         jobs = rpc.get_jobs()
         if hasattr(rpc, "errors"):
             self.errors = rpc.errors
+        return jobs
```

Comment:
While I agree with your assessment that the new naming is more consistent, we do have existing users of this code and I don't think we should change the interface on them.

Reply:
fair enough.  alternate ideas?

Perhaps we add `jobsinfo()` but leaves `jobs()` as a legacy interface?  (unclear how standard the `@deprecated` decorator is at the moment).  Then in the case when the caller whats the raw jobs data .. or `jobdicts()`?

Reply:
I think fetching the raw list of jobs is not going to be a common case. How about we leave the interface as is for now and just fix `flux pstree`